css/style.css

```diff
@@ -0,0 +1,128 @@
+html,
+body {
+  min-width: 290px;
+  color: #4d4e53;
+  background-color: #fff;
+  font-family: "Open Sans", Arial, sans-serif;
+  line-height: 1.5;
+}
+
+#navbar {
+  position: fixed;
+  min-width: 290px;
+  top: 0;
+  left: 0;
+  width: 300px;
+  height: 100%;
+  border-right: solid;
+  border-color: rgba(0, 22, 22, 0.4);
+}
+
+header {
+  color: black;
+  margin: 10px;
+  text-align: center;
+  font-size: 4px;
+  font-weight: bold;
+}
+
+#main-doc header {
+  text-align: left;
+  margin: 0;
+}
+
+section li {
+  margin: 15px 0 0 20px;
+}
+
```

Comment:
Section li selector moved up to the navbar ul

Reply:
it is better to always start with courteous words like please or kindly. know that it is not an order that you know rather it is an orientation. above all, you have to make understand the reason that makes you ask for the change